### PR TITLE
Ui instruction leaderboard

### PR DESCRIPTION
**Pull request by jack-y-xu**
image
image
image

### CODE REVIEW

**Review by asam01 — APPROVED** [review]
lgtm!

File: src/Components/quiz/Instructions.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React, {Component,Fragment} from 'react';
+import {Helmet} from 'react-helmet';
+import {Link, withRouter} from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import * as styles from '../styles/Instructions.jsx';
+
+class Instructions extends Component {
```
Comment (yukyhaha) [inline src/Components/quiz/Instructions.jsx:15]: add comment for what this class is about

File: src/Components/quiz/Instructions.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            - 'play' -> for players to read (in play page)
+            - 'create' -> for creating the room
+     */
+    constructor(props) {
+        super(props);
+        this.state = {
+        }
+    }
+
+    // TODO add more instructions
```
Comment (yukyhaha) [inline src/Components/quiz/Instructions.jsx]: change this line to: TODO(@[username]): add more instructions

File: src/Components/quiz/play/Leaderboard.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, {Component,Fragment} from 'react';
+
+import {firestore} from '../../../../../integrity-capstone-temp/src/firebase';
+import {getUserID} from '../../../../../integrity-capstone-temp/src/firebase';
+import {makeInvestment} from '../../../../../integrity-capstone-temp/src/Components/firebase-access';
+
+class Leaderboard extends Component {
```
Comment (yukyhaha) [inline src/Components/quiz/play/Leaderboard.jsx:12]: Same here, add some comment about this class

**Review by yukyhaha — APPROVED** [review]
Please resolve the comments before merge

File: src/Components/pages/Create.jsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import React, { Component } from 'react';
-import { Link,Redirect } from 'react-router-dom';
-import {db, fire, auth, firestore} from '../../firebase.js';
-import Config from '../pages/Config.jsx';
+import React, {Component} from 'react';
+import {Link,Redirect} from 'react-router-dom';
+import {db,fire,auth,firestore} from '../../firebase.js';
+// import Config from '../pages/Config.jsx';
```
Comment (hairong1007) [inline src/Components/pages/Create.jsx]: Do you want to remove this line?

**Review by hairong1007 — APPROVED** [review]

File: src/Components/pages/Create.jsx (hunk 4 above)
Comment (asam01) [reply src/Components/pages/Create.jsx]: removed!

**Pull request merged**